akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/TypedWatchingClassicSpec.scala

```diff
@@ -39,8 +39,8 @@ object TypedWatchingClassicSpec {
         classic.tell(Typed.Ping(context.self), context.self.toClassic)
 
         Behaviors
-          .receivePartial[Command] {
-            case (context, Pong) =>
+          .receiveMessagePartial[Command] {
+            case Pong =>
```

Comment:
This is the new same name used in surrounding scope warning (`context` in this case)